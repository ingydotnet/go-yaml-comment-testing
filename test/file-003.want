kind: Document
foot: '# foot 7'
content:
    - kind: Mapping
      content:
        - kind: Scalar
          head: |-
            # Show all the things

            # head 1
          line: '# line 2'
          text: foo
        - kind: Sequence
          content:
            - kind: Scalar
              head: '# head 3'
              line: '# line 4'
              text: one
            - kind: Scalar
              style: Single
              head: '# head 5'
              line: '# line 6'
              text: two
            - kind: Scalar
              style: Double
              text: three
            - kind: Scalar
              style: Literal
              text: four
            - kind: Scalar
              style: Folded
              text: five
kind: Document
content:
    - kind: Mapping
      content:
        - kind: Scalar
          head: |-
            # head 8
            # head 9
          text: bar
        - kind: Sequence
          content:
            - kind: Scalar
              tag: '!!int'
              text: "123"
            - kind: Scalar
              tag: '!!float'
              text: "3.1415"
            - kind: Scalar
              tag: '!!bool'
              text: "true"
            - kind: Scalar
              tag: '!!null'
              text: "null"
            - kind: Sequence
              style: Flow
              content:
                - kind: Scalar
                  text: a
                - kind: Scalar
                  text: b
                - kind: Scalar
                  text: c
            - kind: Mapping
              style: Flow
              content:
                - kind: Scalar
                  text: a
                - kind: Scalar
                  tag: '!!int'
                  text: "1"
                - kind: Scalar
                  text: b
                - kind: Scalar
                  tag: '!!int'
                  text: "2"
