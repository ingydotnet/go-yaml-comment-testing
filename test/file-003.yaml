# Show all the things

---
# head 1
foo: # line 2
# head 3
- one # line 4
# head 5
- 'two' # line 6
- "three"
- |-
  four
- >-
  five
# foot 7
...
# head 8
---
# head 9
bar:
- 123
- 3.1415
- true
- null
- [a, b, c]
- {a: 1, b: 2}
