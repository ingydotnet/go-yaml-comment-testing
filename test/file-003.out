Kind: Document
Foot: "# foot 7"
More:  
    Kind: Mapping
    More:  
        Kind: Scalar
        Head:
          "# Show all the things
          
          # head 1"
        Line: "# line 2"
        Text: "foo"
        
        Kind: Sequence
        More:  
            Kind: Scalar
            Head: "# head 3"
            Line: "# line 4"
            Text: "one"
            
            Kind: Scalar
            Styl: Single
            Head: "# head 5"
            Line: "# line 6"
            Text: "two"
            
            Kind: Scalar
            Styl: Double
            Text: "three"
            
            Kind: Scalar
            Styl: Literal
            Text: "four"
            
            Kind: Scalar
            Styl: Folded
            Text: "five"
            
Kind: Document
More:  
    Kind: Mapping
    More:  
        Kind: Scalar
        Head:
          "# head 8
          # head 9"
        Text: "bar"
        
        Kind: Sequence
        More:  
            Kind: Scalar
            Tag : !!int
            Text: "123"
            
            Kind: Scalar
            Tag : !!float
            Text: "3.1415"
            
            Kind: Scalar
            Tag : !!bool
            Text: "true"
            
            Kind: Scalar
            Tag : !!null
            Text: "null"
            
            Kind: Sequence
            Styl: Flow
            More:  
                Kind: Scalar
                Text: "a"
                
                Kind: Scalar
                Text: "b"
                
                Kind: Scalar
                Text: "c"
                
            Kind: Mapping
            Styl: Flow
            More:  
                Kind: Scalar
                Text: "a"
                
                Kind: Scalar
                Tag : !!int
                Text: "1"
                
                Kind: Scalar
                Text: "b"
                
                Kind: Scalar
                Tag : !!int
                Text: "2"
                
